test/testConfigure.js

```diff
@@ -754,6 +754,44 @@ describe('Configure', function() {
     });
   });
 
+  it('configures auto-deploy for a custom remote', function() {
+    nockBasicPostAuthFlow();
+    configure();
+
+    childProcess.execSync('git remote add upstream https://github.com/mozilla/oghliner.git');
+
+    return enterUsernamePassword()
+    .then(function() {
+      return await('Repository remote [default: upstream]:');
+    })
+    .then(function() {
+      emit('origin\n');
+    })
+    .then(complete);
+  });
```

Comment:
It'd also be useful to check that the default value is *origin* if there is more than one remote and none of them are called *upstream*.